Documentation/installation/k8s-install-helm.rst

```diff
@@ -86,21 +86,20 @@ Install Cilium
        .. include:: ../installation/requirements-aks.rst
 
        **Install Cilium:**
-
        Deploy Cilium release via Helm:
 
+       .. note::
+
+            Installing Cilium via helm is supported only for AKS BYOCNI cluster and
+            not for Azure CNI Powered by Cilium clusters.
+            
```

Comment:
Looking again at this change, can we keep the note _below_ the command line? It's strange to introduce the command with “Deploy Cilium release via Helm:” but then to have a note between this and the actual command.

The fix would simply consist in un-indenting the note block. so that `.. note::` is indented at the same level as `.. parsed-literal::`.

Reply:
alright